modules/installation-aws-permissions.adoc

```diff
@@ -7,6 +7,11 @@
 [id="installation-aws-permissions_{context}"]
 = Required AWS permissions for the IAM user
 
+[NOTE]
+====
+Your IAM user must have the privilege `tag:GetResources` in the region `us-east-1` to delete the base cluster resources. As part of the AWS API requirement, the `openshift-installer` performs various actions in this region.
```

Comment:
Two questions:

1) Here, is the word "privilege" different to the word "permission" used in the section below? If they're the same idea, it might be better to use the same word.
2) I wonder whether this information would be better placed (or additionally placed) in the later section about "Supported AWS regions". This note appears to change the status of `us-east-1` from optionally supported to mandatory. Perhaps a note there could both indicate that there was a `us-east-1` requirement and that the user had to have a special permission/privilege?

Reply:
1. +1 "permission" and "privilege" are the same idea so replaced the word with "permission". 
2. Added the note in the "Supported AWS regions" section as well.  I didn't remove the note from the section ` Required AWS permissions for the IAM user ` since SME had suggested placing it there. Agreed. If we place the note additionally in the later section it will benefit the users. 

Addressed